examples/joy-ui-nextjs/src/components/ThemeRegistry/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { extendTheme } from '@mui/joy/styles';
+import { Inter, Source_Code_Pro } from 'next/font/google';
+
+const inter = Inter({
+  subsets: ['latin'],
+  adjustFontFallback: false, // prevent automatically adding Arial as the fallback
```

Comment:
This is different from all the other Material UI demos, which is strange, I would expect the same solution everywhere. Why not go with?

```suggestion
  display: 'swap',
```

Reply:
By default, `display` is already set to `swap`. Do you prefer to set it here explicitly? Or doing another way around, which is to remove `display: swap` from other examples?

Reply:
Right, then maybe?

1. We remove `display: swap` because it's the default
2. We remove `adjustFontFallback` because it creates a layout shift, and also making the assumption that the fallback fonts of Joy UI don't make sense when you correctly configured the font loading.

Reply:
Agree with (1).

For (2), `adjustFontFallback` essentially adds "Arial" or "Times New Roman" as the fallback font depending on whether the font is serif or san serif. I'm not sure using that, instead of Joy UI's fallback is the better idea. IMO, in certain cases, e.g. monospace font, which still makes use of Arial of the fallback, acts really weird.

Reply:
@mwskwong From what I tried, if `fallback` is specified, there is no need to add `adjustFontFallback: false`.

Reply:
@siriwatknp I just tried it again, without setting `adjustFontFallback` to `false` explicitly, the font family will become:

```css
--joy-fontFamily-body: '__Inter_98e1b5','__Inter_Fallback_98e1b5',var(--joy-fontFamily-fallback)
```
In which `__Inter_Fallback_98e1b5` is generated due to `adjustFontFallback` being `true` and is mapped to Arial in this case and takes priority over `var(--joy-fontFamily-fallback)`.

This behavior doesn't seem to be what we are looking for IMO.

Reply:
Rechecked, you are right.